### DISCUSSION TITLE
Feat: Enhancement Categories

### DISCUSSION THREAD

**Discussion by SGFGOV** [medusajs/medusa#3636] [Feature Requests]
Can we add a description field also into the nested Category Entity. It will help with SEO in the front end considerably. 
description can be a simple string.

**Comment by olivermrbl** [comment]
This is an excellent suggestion and requires minimal effort. We will push to add this next week :)

**Reply by SGFGOV** [reply]
Thank you

**Comment by femidof** [comment]
I think adding metadata would be great,the descriptions could be added and if images also want to be added, their url can be used.
